src/shared/containers/WorkSpaceList.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { useAsyncEffect } from 'use-async-effect';
+import { Resource } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+import TabList from '../components/Tabs/TabList';
+import DashBoardList from './DashBoardList';
+
+interface WorkSpaceListProps {
+  workSpaceIds: string[];
+  orgLabel: string;
+  projectLabel: string;
+}
+
+const WorkSpaceList: React.FunctionComponent<WorkSpaceListProps> = ({
```

Comment:
Same comment here, I wouldn't camelcase `WorkSpace` because it makes it look like 2 separate works.

```suggestion
const WorkspaceList: React.FunctionComponent<WorkspaceListProps> = ({
```